core/src/main/java/org/apache/accumulo/core/clientImpl/ClientTabletCacheImpl.java

```diff
@@ -677,7 +678,8 @@ private void requestTabletHosting(ClientContext context,
           extentsToBringOnline.size(), tableId);
       ThriftClientTypes.MANAGER.executeVoid(context,
           client -> client.requestTabletHosting(TraceUtil.traceInfo(), context.rpcCreds(),
-              tableId.canonical(), extentsToBringOnline));
+              tableId.canonical(), extentsToBringOnline),
+          rgid -> rgid.equals(ResourceGroupId.DEFAULT));
```

Comment:
Small syntax improvement:

```suggestion
              tableId.canonical(), extentsToBringOnline), ResourceGroupId.DEFAULT::equals);
```